corehq/apps/case_search/models.py

```diff
@@ -46,6 +45,33 @@ class Meta(object):
         unique_together = ('domain', 'case_type')
 
 
+class RemoveCharacters(models.Model):
```

Comment:
This is a terrible name. I've been having "name block" all day and couldn't come up with anything better...